lib/open_food_network/scope_variant_to_hub.rb

```diff
@@ -21,6 +20,14 @@ def price_in(currency)
         Spree::Price.new(amount: price, currency: currency)
       end
 
+      # Old Spree has the same logic as here and doesn't need this override.
+      # But we need this to use VariantOverrides with Spree 2.0.
+      def in_stock?
+        return true unless Spree::Config[:track_inventory_levels]
+
+        on_demand || (count_on_hand > 0)
```

Comment:
I'm not familiar with variant overrides but I guess `count_on_hand` is a column on a database table added by OFN, right?

Reply:
It's a spree database column on both spree_product and spree_variant.

Reply:
Just a heads-up @mkllnk: `on_demand` is being removed as well in 2.0.

Reply:
@Matt-Yorkley Thanks for the heads up. I wasn't sure if I considered that. But this method is only active if the variant has a VariantOverride. The VariantOverride does have `on_demand` and `count_on_hand`. The vanilla Spree::Variant still has an `in_stock?` method in both Spree versions. So we should be safe to call it.